lib/win-rebase.js

```diff
@@ -0,0 +1,7 @@
+var re = new RegExp(/(.*((node\.exe($| ))|(node($| ))|(iojs($| ))|(iojs\.exe($| )))) ?(.*)$/)
+
+module.exports = function (path, rebase) {
+  var m = path.match(re)
```

Comment:
I generally would use `re.exec(path)` here instead, because it is safer if `path` is not a string - though I admit that "safer" may not actually be safer in all cases.

I'm curious, is there a specific reason for this form? Or is it just what happened to flow from your fingers while typing it up? No wrong answer, I'm honestly curious.